src/components/version-alert-banner/index.tsx

```diff
@@ -19,17 +19,25 @@ import s from './version-alert-banner.module.css'
  */
 export default function VersionAlertBanner({
 	currentVersion,
+	releaseStage,
 	latestVersionUrl,
 }: {
 	currentVersion: string
+	releaseStage?: string
 	latestVersionUrl: string
 }) {
+	let versionMessage = 'You are viewing documentation for version'
+
+	if (typeof releaseStage !== 'undefined' && releaseStage !== 'stable') {
+		versionMessage = 'You are viewing pre-released documentation for version'
```

Comment:
💭 I think this would be correct:
```suggestion
		versionMessage = 'You are viewing pre-release documentation for version'
```

I'm wondering if we want to include the `releaseStage` in the messaging here, or if `pre-release` is enough. What do you think? 🤔

Reply:
Can we use the wording from the [RFC](https://docs.google.com/document/d/1yPrfcQCaEZyyLii8hv-OmoYScqpg94_OAfaFXymo7Y4/edit#)?

> You are viewing documentation for pre-release version {version}. **View latest version.**

![CleanShot 2023-04-14 at 12 27 43@2x](https://user-images.githubusercontent.com/26389321/232102218-0ba20194-c35d-4c5e-8973-b81869ef26fd.png)